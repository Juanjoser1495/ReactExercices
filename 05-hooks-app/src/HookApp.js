export const HookApp = () => {
  return (
    <div>
      <h4>Hola Mundo</h4>
    </div>
  );
};
