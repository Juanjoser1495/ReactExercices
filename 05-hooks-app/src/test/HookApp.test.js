import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe("Pruebas en <HookApp />", () => {
  test("Debe de mostrar el <HookApp> correctamnete", () => {
    const wrapper = shallow(<HookApp></HookApp>);
    expect(wrapper).toMatchSnapshot();
  });
});
