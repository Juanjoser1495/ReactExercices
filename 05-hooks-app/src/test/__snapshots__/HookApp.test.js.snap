// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> Debe de mostrar el <HookApp> correctamnete 1`] = `
<div>
  <h4>
    Hola Mundo
  </h4>
</div>
`;
